R/loadReg2.R

```diff
@@ -444,14 +463,17 @@ predictSolute.loadReg2 <- function(
       }
     }
   }
-  
   # Add dates if requested
   if(date) {
     if(!is.data.frame(preds)) {
       preds <- data.frame(fit=preds)
     }
     # prepend the date column
-    preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #preds <- data.frame(date=getCol(metadata, newdata, "date"), preds)
+    #predLoad returns the dates, don't need to get them from metadata
```

Comment:
Ok, I will leave it then.  I think this could be streamlined in the future though, I still don't see a reason for dropping the dates and then appending them again later.